plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt

```diff
@@ -136,14 +136,14 @@ However, if you still insist on skipping versions, we strongly advise to read al
 * Remove/Rename the file upgrade.properties
 * Restart the server. You're ready.
 
-== Upgrading from Platform 4.4.1 to ${org.exoplatform.platform.version} ==
+== Upgrading from Platform 4.4.2 to ${org.exoplatform.platform.version} ==
 
 === Description of the plugins used in this upgrade ===
 
 # PLF-7088 Problem while naming contents using apostrophes
 commons.upgrade.NodeTypeTemplateUpgradePlugin.enable=true
```

Comment:
Must be updated, PLF-7088 is an issue fixed in 4.4.2. I will check if we need upgrade plugins for 4.4.3.